packages/contracts/contracts/identity/NonFungibleRegistryEnumerableUpgradeable.sol

```diff
@@ -141,7 +141,7 @@ contract NonFungibleRegistryEnumerableUpgradeable is
     event MerkleRootUpdated(bytes32 merkleRoot, bool frozen); 
 
     /// @custom:oz-upgrades-unsafe-allow constructor
-    constructor() initializer {}
```

Comment:
I'm pretty sure this is actually correct for our particular use-case (and it was part of the audited codebase as well). The reason is this: when deployed, we want to make sure that this contract cannot be initialized by anyone else, so we call the `initializer` modifier in the constructor. This isn't a problem for the registries because they are proxies, and thus hold their state separately, allowing them to call the `initialize` function when they are deployed.